.github/workflows/build-doc.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         
       - name: Install sphinx extensions
         shell: bash -l {0}
-        run: conda install -c conda-forge pygments sphinx sphinx-design breathe pydata-sphinx-theme nbsphinx jupytext pandoc ipykernel nbconvert
+        run: conda install -c conda-forge pygments sphinx sphinx-design breathe pydata-sphinx-theme nbsphinx jupytext pandoc ipykernel nbconvert sphinx-panels
```

Comment:
@bonh `sphinx-panels` has been deprecated and should now only be using `sphinx-design`.   Did I miss removing sphinx-panels somewhere?

Reply:
Ah, I didn't know that. Yes, some of the API documentation seems to use panels.

Reply:
But these errors weren't causing the long building time so we can revert that.